compiler/ballerina-parser/src/test/resources/expressions/object-constructor/object_constructor_source_10.bal

```diff
@@ -1,5 +1,5 @@
 public function main() {
     foo(object {
-        int i = 1;
-    });
+                int i = 1;
+            });
```

Comment:
Previous formatting is better in this case. Is there any easy way of special casing these kinds of cases?

Reply:
As discussed offline formatting of function calls with object or record arguments (scoped arguments) is done such that the when there is an object argument the indentation at the beginning of the object is considered as the new indentation for the function call arguments following that argument. The testcase added [here](https://github.com/poorna2152/ballerina-lang/blob/function_call_formatting/misc/formatter/modules/formatter-core/src/test/resources/statements/call/assert/call_statement_5.bal)  demonstrate the such examples